jsonnet/main.jsonnet

```diff
@@ -117,11 +116,17 @@ local inCluster =
         kubeRbacProxyImage: $.values.common.images.kubeRbacProxy,
         promLabelProxyImage: $.values.common.images.promLabelProxy,
       },
-      grafana: {
+      dashboards: {
         namespace: $.values.common.namespace,
-        version: $.values.common.versions.grafana,
-        image: $.values.common.images.grafana,
-        commonLabels+: $.values.common.commonLabels,
+        commonLabels: {
+          [k]: $.values.common.commonLabels[k]
+          for k in std.objectFields($.values.common.commonLabels)
+          // CMO doesn't deploy grafana hence version label not needed anymore
+          if k != 'app.kubernetes.io/version'
+        } + {
+          'app.kubernetes.io/name': 'dashboard',
+          'app.kubernetes.io/component': 'dashboard',
```

Comment:
I'm not sure we need these 2 labels since dashboards are materialized as a configmap consumed by the console and there's no `dashboard` component per se. In summary I would expect the labels to be only:

```
labels:
     app.kubernetes.io/part-of: openshift-monitoring
     console.openshift.io/dashboard: "true"
```

Reply:
I have removed them :)